integration_tests/src/testcases/puppeteer.test.ts

```diff
@@ -102,6 +108,7 @@ describe('Interchain puppeteer', () => {
   });
 
   it('instantiate', async () => {
+    await context.park.restartRelayer('neutron', 1);
```

Comment:
is it a debugging artifact or does it actually provide any value?

Reply:
when a query is deleted, the relayer in most cases exits with the exception about submission data for non-existent query

Reply:
Yes, but you are triggering a restart in a test called 'instantiate', which is ordered before any other test which might delete queries.

Reply:
oh my bad 👍